abstractions/python/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
We do already have a license here https://github.com/microsoft/kiota/blob/main/LICENSE, maybe remove to avoid duplication?

Reply:
This is required since the abstractions package will be hosted on the Python Package Index.